hl/src/H5LT.c

```diff
@@ -1858,49 +1855,24 @@ H5LTset_attribute_double(hid_t loc_id, const char *obj_name, const char *attr_na
 /*-------------------------------------------------------------------------
  * Function: H5LTfind_attribute
  *
- * Purpose: Inquires if an attribute named attr_name exists attached to
- *          the object loc_id.
- *
- * Programmer: Pedro Vicente
- *
- * Date: May 17, 2006
+ * Purpose:  Checks if an attribute named attr_name exists attached to
+ *           the object loc_id
  *
- * Comments:
- *  Calls the private version of the function
+ * TODO:     Overloading herr_t is not a great idea. This function either
+ *           needs to be rewritten to take a Boolean out parameter in
+ *           HDF5 2.0 or possibly even eliminated entirely as it simply
+ *           wraps H5Aexists.
  *
+ * Return:   An htri_t value cast to herr_t
+ *              Exists:         Positive
+ *              Does not exist: 0
+ *              Error:          Negative
```

Comment:
htri_t is an abomination. I'm using how it's described in H5public.h.

Reply:
Yeah, not certain what we were thinking there...